docs/developers/packages/titus-backend/README.md

```diff
@@ -124,19 +144,26 @@ cognito.adminGetUser(params, (err, response) => {
 
 
 
-[Jest]: https://jestjs.io
-[ESLint]: https://eslint.org
-[Prettier]: https://prettier.io
-[Standard]: https://standardjs.com
-[fastify]: https://fastify.io
-[Pino]: http://getpino.io
+
+
+
 [Auth0]: https://auth0.com
-[Nodemon]: https://nodemon.io
+[casbin]: https://casbin.org
+[casbin-introduction]: https://www.nearform.com/blog/access-control-node-js-fastify-and-casbin/
+[cognito]: https://aws.amazon.com/cognito/
+[custom attribute]: https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-attributes.html#user-pool-settings-custom-attributes
+[ESLint]: https://eslint.org
+[faker]: http://marak.github.io/faker.js
 [fastify-postgres]: https://github.com/fastify/fastify-postgres
+[fastify]: https://fastify.io
+[Jest]: https://jestjs.io
 [jwt]: https://jwt.io
 [nock]: https://github.com/nock/nock#readme
-[faker]: http://marak.github.io/faker.js
-[custom attribute]: https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-attributes.html#user-pool-settings-custom-attributes
-[validation instructions]: https://docs.aws.amazon.com/cognito/latest/developerguide/amazon-cognito-user-pools-using-tokens-verifying-a-jwt.html
-[Titus Database Manager]: /developers/packages/titus-db-manager/
 [node-postgres]: https://node-postgres.com/
+[Nodemon]: https://nodemon.io
+[Pino]: http://getpino.io
+[Prettier]: https://prettier.io
+[Standard]: https://standardjs.com
+[Titus Database Manager]: /developers/packages/titus-db-manager/
+[validation instructions]: https://docs.aws.amazon.com/cognito/latest/developerguide/amazon-cognito-user-pools-using-tokens-verifying-a-jwt.html
```

Comment:
do we really have these many new links in this file? it doesn't look like. In general, thought I appreciate you didn't introduce this approach, I prefer inline links because if you put them at the bottom you never know which ones are really used.

Reply:
I agree that inline links are better - I just sorted the list here to make it easier to manage